models/mento/celo/mento_celo_pools.sql

```diff
@@ -0,0 +1,30 @@
+{{
+  config(
+    schema = 'mento_celo',
+    alias = 'pools',
+    materialized = 'view'
+  )
+}}
+
+{% set pool_models = [
+  ref('mento_celo_pools_v1'),
+  ref('mento_celo_pools_v2')
```

Comment:
similar to `dex.trades` lineage, should these be named as below?
`mento_v1_celo_pools`
`mento_v2_celo_pools`

Reply:
would mean changes to file names, config settings, refs, etc etc